.github/workflows/snapshot_tests.yml

```diff
@@ -12,12 +12,10 @@ jobs:
     if: contains(github.event.pull_request.body, 'RUN_SNAPSHOT_TESTS')
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
-        with:
-          submodules: true
       - run: rustup override set ${{ env.RUST_VERSION }}
       - uses: Swatinem/rust-cache@98c8021b550208e191a6a3145459bfc9fb29c4c0 # v2.8.0
-        with:
-          workspaces: ".\nzola" # needed to cache build of zola in ./zola/target
+      - name: Install Zola
```

Comment:
note: it is possible to cache cargo installed binaries using GHA. We could do that to speed up CI.

Probably should try.

ICU4X CI does something similar if you want to have a look.

(can post links later)

Reply:
rust-cache above should already do that.

Reply:
Ah! okay!